DirectProgramming/DPC++FPGA/Tutorials/Features/kernel_args_restrict/README.md

```diff
@@ -130,7 +135,10 @@ When compiling for FPGA hardware, it is recommended to increase the job timeout
      ```
      nmake report
      ``` 
-   * An FPGA hardware target is not provided on Windows*. 
+   * Compile for FPGA hardware (longer compile time, targets FPGA device):
+     ```
+     nmake fpga
+     ``` 
 
 *Note:* The Intel® PAC with Intel Arria® 10 GX FPGA and Intel® FPGA PAC D5005 (with Intel Stratix® 10 SX) do not yet support Windows*. Compiling to FPGA hardware on Windows* requires a third-party or custom Board Support Package (BSP) with Windows* support.
```

Comment:
Should we get rid of the word 'yet' here, while we're at it?  I'm not aware of any plans to ever support Windows with these cards.

Reply:
Me neither. I've removed the "yet".